src/pages/case-study.js

```diff
@@ -70,25 +69,27 @@ export const allCaseStudies = graphql`
         }
       }
     }
-
-    allMarkdownRemark(
-      filter: { frontmatter: { type: { in: ["case-study", "case-study-2"] } } }
-      sort: { fields: [frontmatter___weight], order: ASC }
-    ) {
-      totalCount
-      edges {
-        node {
-          frontmatter {
-            client_name
-            title
-            path
-            tags
-            weight
-            preview_image {
-              childImageSharp {
-                fixed(width: 600, height: 600) {
-                  ...GatsbyImageSharpFixed_withWebp_noBase64
-                }
+    allStrapiCaseStudy {
+      nodes {
+        id
+        Path
+        Style
+        Title
+        Client_Name
+        New_Style {
+          Preview_Image {
+            childImageSharp {
+              fixed(height: 150, width: 150) {
+                ...GatsbyImageSharpFixed_withWebp_noBase64
+              }
+            }
+          }
+        }
+        Old_Style {
+          Preview_Image {
+            childImageSharp {
+              fixed(height: 300, width: 300) {
+                ...GatsbyImageSharpFixed_withWebp_noBase64
               }
```

Comment:
I dont believe it would be easy. It would be possible, the changed needed here is on to change the variables on Strapi's Content Type Builder and then update our queries in the Gatsby code